decidim-blogs/app/views/decidim/blogs/posts/_posts.html.erb

```diff
@@ -1,24 +1,26 @@
-<div class="row small-12">
-  <% paginate_posts.each do |post| %>
-    <div class="card card--post">
-      <% if post.photo.present? %>
-        <%= image_tag post.photo.url, class: "card__image" %>
-      <% end %>
-      <div class="card__content">
-        <div class="card__header">
-          <%= link_to post, class: "card__link" do %>
-          <h2 class="card__title">
-            <%= translated_attribute post.title %>
-          </h2>
-          <% end %>
-          <div class="card__author">
-            <% post_presenter = Decidim::Blogs::PostPresenter.new(post) %>
-            <%= cell "decidim/author", post_presenter.author, from: post, has_actions: true %>
-          </div>
-        </div>
+<% paginate_posts.each do |post| %>
+  <div class="blogs__list-item" data-post>
+    <% if post.photo.present? %>
+      <div class="blogs__list-item__image">
+        <%= image_tag post.photo.url, class: "w-full h-full object-cover" %>
+      </div>
+    <% end %>
+
+    <div class="blogs__list-item__content">
```

Comment:
There are still few problems with the blog post links on the index page:
1. The description does not link anywhere
2. There are multiple links to the same target URL next to each other (as indicated with the red boxes in the image)
![Blog post links](https://user-images.githubusercontent.com/864340/183867340-643212eb-923e-4114-aa9f-c9e165d53e32.png)

I know that in the second point the bottom links are linking to different anchor positions on the page but this is problematic especially for blind keyboard users who will now browse through 4 links to the same target when browsing the site. Please see:
https://www.w3.org/WAI/WCAG21/Techniques/html/H2
https://www.hawaii.edu/access/2019/04/17/19-adjacent-links-used-for-same-destination/

For the first point, it would be important that the whole element is clickable especially for people browsing the site using touch devices. Otherwise they probably think that the site is not working as when they click a blog post they are unable to enter the blog post page (when clicking the description of the post). This is just a general usability issue but it should be fixed by following the accessibility guidelines as stated in the second point.

Reply:
In the new design, the item's metadata (author, endorsements, comments) are not links, so we'll remove them and make the entire item clickable.

Reply:
Addressed to https://github.com/decidim/decidim/issues/9671. It's not straight-forward